src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck do his work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stop his work");
+    }
+}
+
```

Comment:
redundant empty line

Reply:
Good evening, without an empty line at the end of the class, it gives me an error when using Maven clean package: File does not end with a newline. [NewlineAtEndOfFile]